plugins/system76-launch/fu-plugin-system76-launch.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2019 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2021 Jeremy Soller <jeremy@system76.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-hash.h"
```

Comment:
only need `fu-plugin-vfuncs.h` with master

Reply:
If I remove #include "fu-hash.h" it fails to build with FU_BUILD_HASH undeclared

Reply:
I see, I rebased on master and can remove it now.